src/parser.ts

```diff
@@ -19,6 +19,7 @@ export class MessageParser {
         let results = [];
         let lines = data.split('\n');
```

Comment:
I'd say this should be changed to `.split(/\r?\n/)` instead, since then any whitespace can be used later if needed.